packages/@snowpack/plugin-build-script/plugin.ts

```diff
@@ -1,21 +1,33 @@
 import execa from 'execa';
 import npmRunPath from 'npm-run-path';
 import {promises as fs} from 'fs';
-import {SnowpackPlugin, SnowpackConfig} from '../config';
 const cwd = process.cwd();
 
-export function buildScriptPlugin(
-  _: SnowpackConfig,
-  {
-    input,
-    output,
-    cmd,
-  }: {
+// TODO: import this from snowpack
+interface SnowpackConfig {
```

Comment:
nit: Since these have no connection to the main types, I'd vote to just leave this untyped for now until we can import from Snowpack.

Reply:
So figured out the weirdness: Lerna/Yarn was actually working great; it was TypeScript running within a Lerna child process that was complaining it couldn’t find types, even though to me everything seemed to be symlinked properly. The weirdest part is: if I ran `yarn build` manually inside `packages/snowpack`, it just worked! 🙈 I have a feeling investigating this won’t be worth the payoff, so instead, I decided to follow your advice and just manually declare types within Snowpack (comment below).

TL;DR symlinked types for local packages sucks; implemented workaround for now.